test/e2e/lib/check.sh

```diff
@@ -4,6 +4,12 @@
 # matches that of exp_bundles_list, and that all bundle images are pullable.
 function check_bundles() {
   local catalog_image="${1:?catalog image required}"
+
+  if [ "$(arch)" != "x86_64" ]
+  then
+    catalog_image="quay.io/operator-framework/opm:v1.27.1"
```

Comment:
why is the catalog image pointing to the opm image for multi-arch ?

Reply:
I ran through the code, that part is going to come out. It's using a single arch image underneath with a packed opm. The packed opm is being used to do further testing. Once I test with a manifest-list image, it'll be removed.